### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#211414]
Build: https://dev.azure.com/monacotools/a6d41577-0fa3-498e-af22-257312ff0545/_build/results?buildId=270300
Changes: https://github.com/Microsoft/vscode/compare/64b28fb...cc34298

ERROR: ld.so: object '/__w/_temp/codeql3000/distribution/codeql/tools/linux64/${LIB}_${PLATFORM}_trace.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored

**Comment by joaomoreno** [comment]
@bpasero The error you mentioned above was safely ignored.

Correct me if I'm wrong but didn't the pipeline first fail due to Smoke Tests on macOS? https://dev.azure.com/monacotools/Monaco/_build/results?buildId=270300&view=logs&j=a5d99aad-4929-5289-bf4a-26b78032353d&s=a56e7651-4213-56af-9488-c25fe636c10c&t=fb020c41-02a6-539b-31c1-63d55e575c1b&l=145

```
  1) VSCode Smoke Tests (Electron)
       Extensions
         install and enable vscode-smoketest-check extension:
     Error: Timeout of 120000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/runner/work/1/s/test/smoke/out/main.js)
      at listOnTimeout (node:internal/timers:569:17)
      at process.processTimers (node:internal/timers:512:7)
```

**Comment by bpasero** [comment]
Yeah and I see a crash dump reported, filing a new issue for that.

**Comment by joaomoreno** [comment]
So let's use this issue for the flakiness in the `install and enable vscode-smoketest-check extension` test.

**Comment by rzhao271** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/209870

**Comment by sandy081** [comment]
I believe there is a network issue on the machine - fetching extensions from the Marketplace is taking too long.

![Image](https://github.com/microsoft/vscode/assets/10746682/557fbd81-03fd-4c59-88f2-6a9edea6c995)
